content/go/concepts/strings/terms/toupper/toupper.md

```diff
@@ -0,0 +1,112 @@
+---
+Title: 'toupper()'
+Description: 'Returns a lowercase string converted to uppercase.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`toupper()`** function is used to convert lowercase alphabet to uppercase.
+
+## Syntax
+
+```pseudo
+func ToUpper(str string) string
+```
+
+Used to convert a given string to uppercase. 
+The function accepts the given string as argument and returns the final string converting it to uppercase.
+
+> **Note:** This only works with thte standardized alphabet characters.
+
+## Example
+
+In the example below, the whole string "Codecademy" is converted to uppercase.
+
+```go
+package main
+import (
+   "fmt"
+   "unicode"
+)
+
+func ToUpper(s string) string {
+   a := []rune(s)
+   for i, c := range a {
+      if unicode.IsLower(c) {
+         a[i] = unicode.ToUpper(c)
+      }
+   }
+   return string(a)
+}
+
+
+
+func main() {
+   var input string = "Codecademy"
+   var output string = ToUpper(input)
+   fmt.Println(output)
+}
+```
+
+This example results in the following output:
+
+```shell
+CODECADEMY
+```
+
+This can also be acomplshied using the ToTitle() function.
```

Comment:
This second example is pretty much the same as the codebyte. We can get rid of it and just leave the codebyte.